pkgs/development/python-modules/pyqt5-stubs/default.nix

```diff
@@ -0,0 +1,77 @@
+{ python }:
+
+let
+  python' = python.override {
```

Comment:
I don't think this is allowed in pythonPackages

Reply:
You're right. This will probably lead to collisions if someone mixed python3Packages.pyqt5-stubs with a pyqt5 from another dependency. I will move pyqt5-stubs into the (to be created in a new PR) package for nitrokey-app2, so that it doesn't pollute python3Packages.